controllers/pipeline/controller.go

```diff
@@ -419,13 +415,25 @@ func buildVertices(pl *dfv1.Pipeline) map[string]dfv1.Vertex {
 		}
 		vCopy := v.DeepCopy()
 		copyLimits(pl, vCopy)
+		replicas := int32(1)
+		if pl.Status.Phase == dfv1.PipelinePhasePaused {
+			replicas = int32(0)
+		} else {
+			x := vCopy.Scale
+			if x.Min != nil && *x.Min > 1 && replicas < *x.Min {
+				replicas = *x.Min
+			}
+			if x.Max != nil && *x.Max > 1 && replicas > *x.Max {
```

Comment:
so only `Min` is honored as of today since `Max` does not make sense without auto-scaling. right?

what will happen is someone runs `k scale vtx ....` ?

Reply:
Today, `scale vtx` will still be honored, but with auto-scaling, it will not - unless we allow some kind of rules, e.g. scale to a number which is out of min <-> max.